src/containers/PageConstructor/Provider.tsx

```diff
@@ -13,16 +13,16 @@ import {
     ProjectSettingsContextProps,
 } from '../../context/projectSettingsContext';
 import {SSRContext, SSRContextProps} from '../../context/ssrContext';
-import {ConstructorTheme, ThemeValueContext} from '../../context/theme/ThemeValueContext';
-import {WithChildren} from '../../models';
+import {ThemeContext} from '../../context/theme';
+import {Theme, WithChildren} from '../../models';
 
 export interface PageConstructorProviderProps {
     isMobile?: boolean;
     locale?: LocaleContextProps;
     location?: LocationContextProps;
     metrika?: MetrikaContextProps;
     ssrConfig?: SSRContextProps;
-    theme?: ConstructorTheme;
+    theme?: Theme;
```

Comment:
if `<PageConstructorProvider theme={...}>` is used somwhere it could be a braking change

Reply:
Yeah, the types are different. But enum is much better I think. Maybe one should move this PR to next major?